src/js/angular/core/directives.js

```diff
@@ -517,6 +517,8 @@ function searchResourceInput($location, toastr, ClassInstanceDetailsService, Aut
                     } else {
                         $scope.searchRdfResourceByEvent(event.target.value, event);
                     }
+                } else if (event.keyCode === 27 && $location.url() === '/') {
```

Comment:
I don't like this here. Wouldn't it be better to be in the home controller? Listen for it and prevent propagation.

Reply:
agree